common/Cargo.toml

```diff
@@ -15,7 +15,7 @@ itertools = "0.14.0"
 tracing = { workspace = true }
 tracing-log = { workspace = true }
 
-pod2_onchain = { git = "https://github.com/0xPARC/pod2-onchain.git", rev = "ff5cb1655cd51fc842da54e552631d4a781f8e16", default-features=false, features = ["disk_cache"]}
+pod2_onchain = { git = "https://github.com/0xPARC/pod2-onchain.git", rev = "2d7d7d3013f1f53ee53c69ab24ce960891053ffe", default-features=false, features = ["disk_cache"]}
```

Comment:
I noticed that this commit doesn't contain the fixed circuit with the `private` tag on the input proof.

Reply:
Sorry, I got confused with the chronological order of commits and my comment doesn't apply because `2d7d7d3013f1f53ee53c69ab24ce960891053ffe` came after https://github.com/0xPARC/blob-e2e-poc/pull/22/commits/8cf6c95f26cf43cfc460a9b6d1f18c600f260135 which applies the fix.

Reply:
it does, no? the `2d7d7d3...` is the latest commit from the PR https://github.com/0xPARC/pod2-onchain/pull/5/commits , which uses the gnark-plonky2-verifier version `8fe52b5...` which has the `private` tag: https://github.com/arnaucube/gnark-plonky2-verifier/commit/8fe52b5161d8e48913e804dc02aa066fa1092285

Reply:
oh I see now your 2nd message (after refreshing the web view), cool!

Reply:
In any case, probably we can first merge the https://github.com/0xPARC/pod2-onchain/pull/5 and the update this current PR to point to the latest (new) main branch of pod2-onchain